packages/v4/patternfly-docs/content/design-guidelines/components/progress/progress.md

```diff
@@ -44,8 +44,6 @@ Use a time interval value such as “4 minutes remaining” to communicate progr
 
 Avoid using indeterminate progress bars to communicate progress that can’t be measured. To track progress that can’t be quantified with a percentage, time, or step, use a [spinner](/components/spinner) instead.
```

Comment:
Would restructure the whole "Usage" section of these guidelines to follow the "When to use" H3 structure with bullets underneath...